src/Sylius/Bundle/CoreBundle/Fixture/Factory/AdminUserExampleFactory.php

```diff
@@ -32,15 +36,28 @@ class AdminUserExampleFactory extends AbstractExampleFactory implements ExampleF
     /** @var string */
     private $localeCode;
 
-    public function __construct(FactoryInterface $userFactory, string $localeCode)
-    {
+    /** @var FileLocatorInterface */
+    private $fileLocator;
+
+    /** @var ImageUploaderInterface */
+    private $imageUploader;
+
+    public function __construct(
+        FactoryInterface $userFactory,
+        string $localeCode,
+        FileLocatorInterface $fileLocator,
```

Comment:
🚨 BC ALERT 🚨

New dependencies should be optional to keep backward compatibility. If not provided, then we should trigger error to inform, that it should be added